### ISSUE TITLE
Imported Bullet Dynamic Rigid Bodies Z and Y Coordinates are Slighly Off

### ISSUE THREAD

**Issue by woofers** [woofers/woofers3d#10]
This is due to Blender's Z up Cartesian coordinate system.

**Comment by woofers** [comment]
https://github.com/woofers/woofers3d/commit/6e204c36cfe8aebcda79792ef3f1e8af75736eb1

**Comment by woofers** [comment]
Models are still slightly displaced as a result of this.

**Comment by woofers** [comment]
The conversion between Y and Z is being done correctly and works perfectly on static bodies.  For some reason the `btBulletWorldImporter` and Blender is using a slightly different point of origin to track Z and Y only on rigid bodies.

**Comment by woofers** [comment]
Turns out the problem is due how I was exporting the `.bullet` files from the Blender game engine.  As recommended by most people on the Blender forums I hooked it up to a key press.  When the key is press it exports the location and physics data at that time in the simulation.  Since I was not pressing the key at the exact start of the scene, it was exporting data slightly after the objects had fallen.  To prevent this from happening, I changed to `.bullet` export script to execute at the start up of the scene.
![blender](https://user-images.githubusercontent.com/7284672/36115716-c66bcb16-0fe8-11e8-9477-ad386e264752.PNG)
